bin/disco_aws.py

```diff
@@ -226,6 +226,36 @@ def get_preferred_private_ip(instance):
         return interfaces[1].private_ip_address
 
 
+def format_listhosts_header(args, most):
+    line = u"{0:<11} {1:<25} {2:<15}".format('ID', 'Hostclass', 'IP')
```

Comment:
I kind of want to extract all these copy-pasted format strings to constants.  Somebody explain to me why that's a bad idea...